Update Body Text/Heading - Rancher backup-restore-operator Default Storage Location (#1106)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md

## Default Storage Location Configuration

> Configure a storage location where all backups are saved by default. Various configuration options are available, including specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` helm chart. You will also have the option to override the configured storage location with each backup, but will be limited to using an S3-compatible or Minio object store.

Reviewer: ```suggestion
Configure a default storage location for your backups. There are various configuration options, such as specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` Helm chart. You also have the option to override the configured storage location with each backup, but are limited to using an S3-compatible or Minio object store.
```

Avoiding passive voice and future tense when not necessary

Author: Thanks for the review Marty! Pushed the changes to all versions and will make a note to maintain present tense in the future.

For information on configuring these options, refer to [this page.](../../../reference-guides/backup-restore-configuration/storage-configuration.md)

### Example YAML File: Rancher Backup Helm Chart

> The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-valuesyaml-for-the-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.

Reviewer 2: ```suggestion
The example [values.yaml file](../../../reference-guides/backup-restore-configuration/storage-configuration.md#example-yaml-file-rancher-backup-helm-chart) can be used to configure the `backup-restore-operator` when the Helm CLI is used to install it.
```
